cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/ReviewResult.java

```diff
@@ -43,12 +43,12 @@ public ReviewResult(Map<Integer, RequestInfo> requestInfoById, Set<Integer> filt
   private String getPlaintext() {
     StringBuilder sb = new StringBuilder();
     int padding = 2;
-    int idLabelSize = ID.length();
-    int submitterAddressLabelSize = SUBMITTER_ADDRESS.length();
+    int idLabelSize = 2;
+    int submitterAddressLabelSize = 17;
     int submissionTimeLabelSize = 15; // Plaintext response, returns submission time -- i.e. not submission time ms.
-    int statusLabelSize = STATUS.length();
-    int endpointWithParamsLabelSize = ENDPOINT_WITH_PARAMS.length();
-    int reasonLabelSize = REASON.length();
+    int statusLabelSize = 6;
+    int endpointWithParamsLabelSize = 20;
+    int reasonLabelSize = 6;
```

Comment:
Nit: I think it is valuable to keep the `.length()` as before rather than hard-coding the numbers. You may add `+ 1` for space when needed and add a comment to indicate why we do so.